src/main/java/mesosphere/dcos/client/DCOSAPIInterceptor.java

```diff
@@ -16,9 +24,9 @@ public void apply(RequestTemplate template) {
         }
 
         if (apiSources.iterator().next().equals(HeaderUtils.MARATHON_API_SOURCE)) {
-            template.insert(0, "/marathon");
+            template.insert(0, marathonPath);
```

Comment:
Is it a problem that the new default is `/service/marathon` but this was just `/marathon`?

Reply:
No, I tested this and both are available in 1.10 AND 1.9 clusters. Just figured I'd make it more consistent.